.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -74,7 +74,7 @@ fun BuildSteps.substDirOnWindows(os: Os) {
         script {
             name = "SETUP_VIRTUAL_DISK_FOR_PERF_TEST"
             executionMode = BuildStep.ExecutionMode.ALWAYS
-            scriptContent = """subst p: "%teamcity.build.checkoutDir%" """
+            scriptContent = """dir p: || subst p: "%teamcity.build.checkoutDir%" """
```

Comment:
I think if it exists, we should remove the drive instead of assuming that it points to the right location.